spec/ddtrace/tracer_integration_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
 
 require 'ddtrace'
-require 'ddtrace/ext/runtime'
-require 'ddtrace/runtime/identity'
+require 'datadog/core/environment/ext'
+require 'datadog/core/environment/identity'
```

Comment:
Minor: After `require 'ddtrace'`, the extra requires are kinda useless ;) . Also, there's no references to `Datadog::Core` anyway on this file.

Reply:
A legacy of simpler times, removed everything except `require 'ddtrace'`, given this is an integration test.